brainstormforce/ultimate-addons-for-gutenberg
# fix: Added Updated SVG's for Columns Block, Added some webpack configs & Updated the Image for UAG Dashboard.

### Description
fix: Added Updated SVG's for Columns Block, Added some webpack configs & Updated the Image for UAG Dashboard

### Types of changes
Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
- Check Columns Block SVG's
- Check UAG Dashboard Image

### Checklist:
